integrationtests/src/test/java/org/neo4j/procedure/ProcedureIT.java

```diff
@@ -362,6 +363,22 @@ public void shouldNotBeAbleToCallWriteProcedureThroughReadProcedure() throws Thr
     }
 
     @Test
+    public void shouldNotBeAbleToCallReadProcedureThroughWriteProcedureInWriteOnlyTransaction() throws Throwable
```

Comment:
There are other existing tests testing that you can call nested procedures, e.g. `shouldBeAbleToCallWriteProcedureThroughWriteProcedure` and `shouldCallRecursiveProcedure`. Do you think it is necessary for this specific enumeration?